PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/TransparencyController.rbxmx

```diff
@@ -137,7 +137,13 @@ local function CreateTransparencyController()
 	end
 
 	function module:SetSubject(subject)
-		local character = subject and subject:IsA('Humanoid') and subject.Parent
+		local character = nil
+		if subject and subject:IsA("Humanoid") then
+			character = subject.Parent
+		end
+		if subject and subject:IsA("VehicleSeat") then
+			character = subject.Occupant.Parent
```

Comment:
We need to nil check this: Occupant could be nil and then accessing the Parent property will error the script.